activities/LabyrinthJS.activity/index.html

```diff
@@ -2,65 +2,66 @@
 <html>
 
 <head>
-<meta charset="utf-8" />
-<title>LabyrinthJS Activity</title>
-<meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, viewport-fit=cover"/>
-<link rel="prefetch" type="application/l10n" href="locale.ini" />
-<link rel="stylesheet" media="not screen and (device-width: 1200px) and (device-height: 900px)"
-  href="lib/sugar-web/graphics/css/sugar-96dpi.css">
+  <meta charset="utf-8" />
```

Comment:
Please do not change indentation style. Only updated lines should be changed in your PR.
It's probably due to options in your IDE.

Reply:
Sorry my bad, will be fixing it.